localization/yabloc/yabloc_monitor/src/availability_module.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 TIER IV
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "availability_module.hpp"
+
+#include <rclcpp/logging.hpp>
+
+#include <memory>
+
+using PoseStamped = geometry_msgs::msg::PoseStamped;
```

Comment:
Line 28 of availability_module.hpp is duplicated.

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/4395/commits/d94f5639cfc095bf9e07cb7706e504b1a7bf29e3 Fixed!